test/preprocessor.cpp

```diff
@@ -0,0 +1,34 @@
+#include <catch.hpp>
+#include <fstream>
+
+#include "libclang/preprocessor.hpp"
+#include "test_parser.hpp"
+
+using namespace cppast;
+
+TEST_CASE("preprocessor_parses_escaped_character")
```

Comment:
Tests are failing on old libclang where I can't handle includes. Could you add `"[!hide][clang4]"` tags to the test case?

Reply:
Done :)